api/tests/team/acceptance/application/membership/membership.route.test.js

```diff
@@ -406,4 +406,34 @@ describe('Acceptance | Team | Application | Route | membership', function () {
       });
     });
   });
+
+  describe('PATCH /api/organizations/{id}/me', function () {
+    context('when user is one of the members of the organization', function () {
+      it('updates user membership last accessed at', async function () {
```

Comment:
Dans le code on va plutôt chercher cette information en tant que « lastAccessedAt » : 
```suggestion
      it('updates user membership lastAccessedAt', async function () {
```